### PR TITLE
Inline template creation

### PR DESCRIPTION
**Pull request by clabland**
Don't rely on `html-loader`, simply inline the templates.

Certain tooling setups, such as `pnpm`, would fail to build: https://github.com/customerio/gist-web/issues/44

### CODE REVIEW

File: src/templates/embed.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                height: 100%;
+                width: 100%;
+            }
+            #gist-embed-container .gist-frame {
+                height: 100%;
+                width: 100%;
+                border: none;
+            }
+            @media (max-width: ${maxWidthBreakpoint}px) {
+                #x-gist-bottom, #x-gist-bottom, #x-gist-floating-top, #x-gist-floating-bottom, #x-gist-floating-top-left, #x-gist-floating-top-right, #x-gist-floating-bottom-left, #x-gist-floating-bottom-right {
```
Comment (jatinn) [inline src/templates/embed.js:51]: `#x-gist-bottom` listed twice was like that in the old code as well but wonder if thats expected/needed or if one is suppose to be `#x-gist-top` instead

File: src/templates/embed.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+export function embedHTMLTemplate(messageProperties, url) {
+    const wideOverlayPositions = ["x-gist-bottom", "x-gist-bottom", "x-gist-floating-top", "x-gist-floating-bottom"];
```
Comment (jatinn) [inline src/templates/embed.js:2]: similar `x-gist-bottom` listed twice in the list

File: src/templates/embed.js (hunk 1 above)
Comment (jatinn) [inline src/templates/embed.js:51]: `x-gist-bottom` listed twice, not sure if needed or if one is suppose to be `x-gist-top` instead.
seems like existed in the old file as well so not a port issue

**Review by jatinn — APPROVED** [review]

File: src/templates/embed.js (hunk 1 above)
Comment (BernardGatt) [reply src/templates/embed.js]: Nice catch @jatinn

**Pull request merged**